tests/Services/Search/ParametersHandlerTest.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+namespace Findologic\Tests\Services\Search;
+
+use Ceres\Helper\ExternalSearchOptions;
+use Findologic\Services\Search\ParametersHandler;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+use Plenty\Plugin\Http\Request;
+
+/**
+ * Class SearchServiceTest
+ * @package Findologic\Tests
+ */
+class ParametersHandlerTest extends TestCase
+{
+    public function providerGetSortingOptions()
+    {
+        return [
+            '$isSearch is false' => [
```

Comment:
Data set descriptions should be in "clear text" since you want to describe scenarios rather than technical details:
* Sorting options for category page
* Sorting options for search page
* Force option "Relevance" for search page